packages/components/addon/components/hds/accordion/row/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class HdsAccordionRowIndexComponent extends Component {
+  @tracked isOpen = this.args.isOpen ?? false;
+
+  @action
+  toggle() {
+    this.isOpen = !this.isOpen;
+  }
```

Comment:
I added this to be able to set a "hds-accordion-row--is-open" class name on the component parent for more flexibility in styling the open vs. closed states.